src/shared/models/friends.model.ts

```diff
@@ -0,0 +1,8 @@
+
+import type { User } from './user.model.js';
+export interface Friends {
+    _id?:string;
+    user: User;
+    friends:[]
```

Comment:
```suggestion
    friends: string[]
```
for extra typing and accuracy this should probably say an array of strings(array of _ids)